libgnucash/backend/xml/gnc-xml-backend.cpp

```diff
@@ -149,7 +149,7 @@ GncXmlBackend::session_begin(QofSession* session, const char* new_uri,
 
     /* Set the lock file */
     m_lockfile = m_fullpath + ".LCK";
-    auto locked = get_file_lock();
+    auto locked = get_file_lock(mode == SESSION_BREAK_LOCK);
```

Comment:
Could this be `const auto`?

Reply:
Yes but I wouldn't normally litter the code with extra `const` specifiers.